pexec/managed_process_test.go

```diff
@@ -581,6 +582,38 @@ done`, tempFile.Name()))
 	})
 }
 
+func TestManagedProcessEnvironmentVariables(t *testing.T) {
+	t.Run("set an environment variable", func(t *testing.T) {
+		logger := golog.NewTestLogger(t)
+		output := new(bytes.Buffer)
+		proc := NewManagedProcess(ProcessConfig{
+			Name:                 "bash",
+			Args:                 []string{"-c", "printf %s $VIAM_HOME"},
```

Comment:
What's the `%s` here?

Reply:
%s for c style (printf) but I just realized I should swap to `printenv`. Will do that.

Reply:
Ah cool, I thought it was a vestigial go formatting directive.

Reply:
Resolving in favor of `printenv`